api/api/models/image.py

```diff
@@ -118,6 +143,26 @@ class ImageReport(AbstractMediaReport):
         help_text="The reference to the image being reported.",
     )
 
+    @property
+    def image_url(self):
+        return super().url("images")
+
+
+class NsfwReport(AbstractMediaReport):
```

Comment:
I know the IP says to use `NsfwReport` for this but I think it would be clearer to use `NsfwImageReport` instead for the temporary name, like we do for the other media-specific models.